### ISSUE TITLE
Swift Package Manager support

### ISSUE THREAD

**Issue by richardtop** [grame-cncm/libmusicxml#62]

**Comment by arshiacont** [comment]
Hi Richard,

LibMusicXml is a C++ library and open source. You can directly make a framework and embed in it in your Swift project or make a Swift Package (using compiled binary).

**Comment by richardtop** [comment]
Hi, I'm interested in making this library SPM-compatible with source, i.e. without the need to precompile it.

I'll make a PR if I could make it work 😀

**Comment by arshiacont** [comment]
Interesting.. using new C++ interop I suppose.. Looking forward.

**Comment by richardtop** [comment]
Not necessarily, we can start with just being able to build it and connect it with the SPM, so that it would be possible to call the library from some other C++ code. And that code could use some interop approaches (C, or C++, maybe Objective-C++) to provide an interface to the higher layer.

Being able to build the project with the SPM would make it more accessible for Apple developers.
